File: csharp/dsf/dsf/src/route/NoUseableServiceException.cs
using System;

namespace com.baidu.dsf.route
{

	/// 
	/// <summary>
	/// @author xiaohaixing
	/// </summary>
	public class NoUseableServiceException : Exception
	{

		public NoUseableServiceException(string msg) : base(msg)
		{
		}

		public NoUseableServiceException(string msg, Exception ex) : base(msg, ex)
		{
		}
	}

}

File: java/dsframework/src/main/java/dsf/route/NoUseableServiceException.java
package dsf.route;

/**
 *
 * @author xiaohaixing
 */
public class NoUseableServiceException extends RuntimeException {

    public NoUseableServiceException(String msg) {
        super(msg);
    }

    public NoUseableServiceException(String msg, Throwable ex) {
        super(msg, ex);
    }
}
